edweiss-app/model/lectures/lectureDoc.ts

```diff
@@ -38,8 +47,11 @@ namespace LectureDisplay {
 	}
 
 	export const Functions = FunctionFolder("lectures", {
-		addAudioTranscript: FunctionOf<{ courseId: string, lectureId: string, pageNumber: number, transcription: string; }, 'successfully_added', 'invalid_arg' | 'error_firebase' | 'course_not_found'>("addAudioTranscript"),
+		addAudioTranscript: FunctionOf<{ courseId: string, lectureId: string, pageNumber: number, transcription: string; }, {}, 'invalid_arg' | 'error_firebase' | 'successfully_added'>("addAudioTranscript"),
+		createQuestion: FunctionOf<{ courseId: string, lectureId: string, question: string; }, { id: string; }, 'invalid_arg' | 'error_firebase' | 'empty_question'>("createQuestion"),
```

Comment:
I understand that this could lead into duplication, but I don't see how else to do it, I need to handle these types of errors each time since they are the ones that could happen during execution of the functions